components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -3429,11 +3448,25 @@ std::string RewardsServiceImpl::GetExternalWalletType() const {
           country_codes::CountryCharsToCountryID(country.at(0), country.at(1));
 
       if (id == current_country)
-        return ledger::constant::kWalletBitflyer;
+        return true;
     }
   }
+  return false;
+}
 
+std::string RewardsServiceImpl::GetExternalWalletType() const {
+  if (IsBitFlyerRegion()) {
+    return ledger::constant::kWalletBitflyer;
+  }
+#if defined(OS_ANDROID)
```

Comment:
os guards is not appropriate here, see https://github.com/brave/brave-core/pull/8566/files#r664753766

Reply:
resolved: https://github.com/brave/brave-core/pull/8566/files#diff-b3817ac19a1c6ad3db2832fc878795ccc8a6f5fc0ae281cdf70d85a3a0a3d324R1309